Bottom: 416d62e9d05cddb9f8956568d29685cbd849d40b
Top:    6979d018d06ccb16c61579c284e2fba19834873a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-21 19:37:17 +0000

Refresh of add-support-for-filling-thindr

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 336fae4..d2b680b 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -28,6 +28,7 @@
 
 #include "hid.h"
 #include "../hidint.h"
+#include "hid/common/draw_helpers.h"
 #include "lesstif.h"
 
 #ifdef HAVE_LIBDMALLOC
